src/IEVMBlackjack.sol

```diff
@@ -0,0 +1,112 @@
+// // SPDX-License-Identifier: MIT
+// pragma solidity 0.8.16;
+
+// interface IEVMBlackjack {
```

Comment:
up to have the events, errors, data structures, methods in here. just need to uncomment + update

Reply:
Resovled